spec/openapi.yaml

```diff
@@ -931,45 +658,60 @@ components:
               description: The size of the file in bytes.
               example: 873151
               nullable: true
-        estimatedPrintTime:
-          type: number
-          description: Estimated print time for the file in seconds.
-          nullable: true
         lastPrintTime:
           type: integer
           description: Last print time for the file in seconds.
           nullable: true
-        filament:
-          type: object
-          properties:
-            length:
-              type: number
-              description: Length of filament used in mm.
-              nullable: true
-            volume:
-              type: number
-              description: Volume in filament used in cm3.
-              nullable: true
-          nullable: true
-    OctoProgress:
+
+    Progress:
+      oneOf:
+        - $ref: "#/components/schemas/ProgressMini"
+        - $ref: "#/components/schemas/ProgressSL1"
       type: object
       properties:
         completion:
           type: number
           description: Percentage of completion of the current job.
           nullable: true
+          example: 55.6349010
         filepos:
```

Comment:
Could we ignore filepos?

Reply:
Yep. SL1 can ignore it. Maybe FDM also but I cannot make decisions for FDM